values.yaml

```diff
@@ -16,6 +16,21 @@ deploy:
   container:
     extraJvmOpts: ""
     libraries: ""
+    # [USER] Define custom environment variables using standard K8s format
+    # env:
+    #  - name: STANDARD_KEY
+    #    value: standard value
+    #  - name: CONFIG_MAP_KEY
+    #    valueFrom:
+    #      configMapKeyRef:
+    #        name: special-config
+    #        key: special.how
+    #  - name: SECRET_KEY
+    #    valueFrom:
+    #      secretKeyRef:
+    #        name: special-secret
+    #        key: special.how
+    extraEnv:
```

Comment:
There's an inconsistency here between the commented example and the default element, one is `env` the other is `extraEnv`. IMO `env` is better as it's more succinct and follows the k8s convention. From the user perspective, they are simply adding their env variables, the fact we add 3 of our own is irrelevant to them, so I don't think the added meaning of `extraEnv` is required.

Reply:
Yeah, I originally had it as `env` and switched it out. Switched it back now.